drivers/misc/cldatadrv/rcar_cluster_drv.c

```diff
@@ -0,0 +1,389 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * rcar_cluster_drv.c  --  R-Car Cluster driver
+ *
+ * Copyright (C) 2019-2023 EPAM Systems GmBh Corporation
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+
+#include <linux/module.h>
+#include <linux/slab.h>
+#include <linux/wait.h>
+#include <linux/miscdevice.h>
+#include <linux/rpmsg.h>
+#include "uapi/linux/r_taurus_cluster_protocol.h"
```

Comment:
`#include <uapi/linux/r_taurus_cluster_protocol.h>`

Reply:
regarding cpp , this is because it took the file from rvgc :( and did not check